.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+# Hey, I just made a Pull Request!
+
+## Description
+<!--- Describe your changes -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## Have you tested this? If so, how?
+<!--- Valid responses are "I have included unit tests." or --> 
+<!--- "I ran my jobs with this code and it works for me." -->
+
+## Checklist for PR author(s)
+<!--- Put an `x` in all the boxes that apply: -->
+- [ ] Changes are covered by unit test
+- [ ] All tests pass
+- [ ] Code coverage check passes
+- [ ] Error handling is tested
+- [ ] Errors are handled at the appropriate layer
+- [ ] Errors that cannot be handled where they occur must be propagated
+- [ ] (optional) Changes are covered by integration test
```

Comment:
We should add styx e2e integration tests so this can be done

Reply:
Hmm, I think I meant system test here.